docs/migration-guides/1.22-1.23.md

```diff
@@ -0,0 +1,43 @@
+# Migrate from 1.22 to 1.23
+
+## Increase the Loki volume size
+In Kyma 1.23.0, the size of the persistent volume claim has been increased from 10Gi to 30Gi
```

Comment:
I'd rephrase it a bit since a PVC doesn't have a size, it requests a volume of a certain size. 

``` suggestion
In Kyma 1.23.0, the Loki volume size requested by the persistent volume claim has been increased from 10Gi to 30Gi
```

Reply:
The same wording ( the PVC size) is used below